### DISCUSSION TITLE
Disable actions per resource

### DISCUSSION THREAD

**Discussion by krystof-k** [avo-hq/avo#2552] [Feature requests]
For example, I'd like to disable create for a resource which should never be created via admin panel (or delete).

I guess it could be possible to it now via Pundit policies, but this should be like a general option per resource.

**Comment by adrianthedev** [comment]
> I guess it could be possible to it now via Pundit policies, but this should be like a general option per resource.

That is the general option per resource.

As Avo is vast and there are multiple options, we want to avoid having two or more ways of doing the same thing.
From experience, we realized that having two or more ways of doing something will lead to edge-cases and a bad experience for users.

For now, we'll keep only the policy ways of hiding/showing buttons.

Thanks for the feedback @krystof-k!
